etc/migration/disable_users_from_tenancy/lib/disable-users-by-tenancy.js

```diff
@@ -0,0 +1,132 @@
+/*!
+* Copyright 2017 Apereo Foundation (AF) Licensed under the
+* Educational Community License, Version 2.0 (the "License"); you may
+* not use this file except in compliance with the License. You may
+* obtain a copy of the License at
+*
+*     http://opensource.org/licenses/ECL-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an "AS IS"
+* BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+* or implied. See the License for the specific language governing
+* permissions and limitations under the License.
+*/
+
+/*
+* Disable users belonging to a disabled tenancy
+* Github issue #1304
+*/
+
+var _ = require('underscore');
+var bunyan = require('bunyan');
+var async = require('async');
```

Comment:
If you want to use `async` so badly, you should probably add it to `package.json`...